omod/src/main/java/org/openmrs/module/cohort/web/resource/CohortMemberResource.java

```diff
@@ -102,24 +102,23 @@ public CohortMember newDelegate() {
 	}
 	
 	@Override
-	public CohortMember save(CohortMember cohortMember) throws ResponseException {
-		CohortM cohort = cohortMember.getCohort();
-		Patient newPatient = cohortMember.getPatient();
+	public CohortMember save(CohortMember newMember) throws ResponseException {
```

Comment:
Will this parameter always be a new member? Don't we have cases where this method is called for existing members?

Reply:
Yes good point, calling it newMember could be confusing. I was trying to distinguish it from the member variable we get from the cohort, so I could change it to "newOrUpdatedMember" or do you have another preference?

Reply:
How about leaving the name as it was before your changes?

Reply:
> How about leaving the name as it was before your changes?

Done - reverted the name change